examples/lightning_base.py

```diff
@@ -96,25 +93,14 @@ def optimizer_step(self, epoch, batch_idx, optimizer, optimizer_idx, second_orde
         optimizer.zero_grad()
         self.lr_scheduler.step()
 
-    def get_tqdm_dict(self):
-        avg_loss = getattr(self.trainer, "avg_loss", 0.0)
-        tqdm_dict = {"loss": "{:.3f}".format(avg_loss), "lr": self.lr_scheduler.get_last_lr()[-1]}
-        return tqdm_dict
-
-    def test_step(self, batch, batch_nb):
-        return self.validation_step(batch, batch_nb)
-
-    def test_end(self, outputs):
-        return self.validation_end(outputs)
-
     def train_dataloader(self):
         train_batch_size = self.hparams.train_batch_size
         dataloader = self.load_dataset("train", train_batch_size)
 
         t_total = (
-            (len(dataloader.dataset) // (train_batch_size * max(1, self.hparams.n_gpu)))
-            // self.hparams.gradient_accumulation_steps
-            * float(self.hparams.num_train_epochs)
+            (len(dataloader.dataset) // (train_batch_size * max(1, self.hparams.gpus)))
```

Comment:
Does this take into consideration TPU cores? Is that something that it needs to do?